src/views/spotlights/housing/dashboard.vue

```diff
@@ -66,16 +66,12 @@
         :init-value="caseControls"
         @selected="updateCaseControls"
       />
-      <div v-if="caseControls.age.value === 'adjusted'">
-        What is an age-adjusted rate? Age plays a big role in a person's risk
-        for COVID-19. Different groups of people have different age
-        distributions - meaning one group may have more old people, or one group
-        may have more young people. Age adjusting makes it so we can compare
-        between groups that have different age distributions.
-      </div>
-      <div v-else-if="caseControls.age.value === 'specific'">
-        Rates tell us how many people have tested positive for COVID-19 compared
-        to a larger group taking into account the population size of that group.
+      <div>
+        {{
+          controls.age.value === "adjusted"
```

Comment:
why did we get rid of the distinction between caseControls and separate controls for hospitalizations?